graphistry/feature_utils.py

```diff
@@ -135,29 +168,32 @@ def assert_imported():
 #
 #      _featurize_or_get_edges_dataframe_if_X_is_None
 
-FeatureEngineConcrete = Literal["none", "pandas", "dirty_cat", "torch"]
+FeatureEngineConcrete = Literal["none", "pandas", "dirty_cat", "torch", "cuCat"]
```

Comment:
lowercase? and wondering if we should do `"cu_cat"` for consistency as well?

Reply:
changed here and graphistry/cu-cat branch